<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegAccMenu</class>
 <widget class="QWidget" name="RegAccMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegAccMenu</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border-image: url(:/StudentScoreManageSys/img/logintest.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/StudentScoreManageSys/img/BackButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>182</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-image: url(:/StudentScoreManageSys/img/white.png);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="checkPassword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>302</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-image: url(:/StudentScoreManageSys/img/white.png);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="pAssword">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>262</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-image: url(:/StudentScoreManageSys/img/white.png);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="accText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>222</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-image: url(:/StudentScoreManageSys/img/white.png);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>26</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>26</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pAsswordLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>26</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkPasswordLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>26</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请重复输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accENDBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StudentScoreManageSys.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>backBtn</sender>
   <signal>clicked()</signal>
   <receiver>RegAccMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
